CRM/Admin/Form/Setting/Localization.php

```diff
@@ -200,7 +198,7 @@ public function postProcess() {
     }
 
     // if we manipulated the language list, return to the localization admin screen
-    $return = (bool) (CRM_Utils_Array::value('makeMultilingual', $values) or CRM_Utils_Array::value('addLanguage', $values));
+    $return = (!empty($values['makeMultilingual']) || !empty($values['addLanguage']));
```

Comment:
Curious why this is even conditional. Why not just always return to the page? It's weird that it doesn't sometimes.